tools/mc-java/src/test/java/io/spine/tools/mc/java/gradle/AnnotatorPluginTest.java

```diff
@@ -99,49 +105,49 @@ void createTempDir() {
         @Test
         @DisplayName("if file option is true")
         void ifFileOptionSet() throws IOException {
-            checkNestedTypesAnnotations(INTERNAL_ALL, true);
+            checkNestedTypesAnnotations(INTERNAL_ALL.fileName(), true);
```

Comment:
Maybe we can just have an override which does this `fileName()` call internally?